graylog2-web-interface/src/components/scratchpad/ScratchpadModal.tsx

```diff
@@ -251,10 +233,7 @@ const Scratchpad = () => {
           </StatusMessage>
 
           <ButtonGroup>
-            <Button data-clipboard-button
-                    data-clipboard-target={`#${TEXTAREA_ID}`}
-                    id="scratchpad-actions"
-                    title="Copy">
+            <Button id="scratchpad-actions" onClick={() => copyCallback()} title="Copy">
```

Comment:
We can just pass the function itself, that avoids creating an anonymous function on each render:

```suggestion
            <Button id="scratchpad-actions" onClick={copyCallback} title="Copy">
```